web/src/ui_report.ts

```diff
@@ -39,6 +39,9 @@ export function error(
     status_box: JQuery,
     remove_after?: number,
 ): void {
+    if (xhr?.status === 400 && status_box.attr("id") === "read_receipts_error") {
+        response_html = "";
+    }
```

Comment:
I think realistically, the client should be able to check whether read receipts are disabled in the organization without asking the server anything, and that would be a better design.

If we did need to identify a specific error code here, though, I think the right model would be to use a `CodedError` in the server. But I don't think that's necessary.